docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design system.</p>
```

Comment:
```suggestion
<p class="description">Material UI is library of React UI components that implements Google's Material Design.</p>
```

Maybe we could ditch the _system_ there? As in "Material Design" is the name of the design system, so it would probably need to be "Material Design design system" 😅 Does it make sense?

Reply:
👍